apps/src/templates/teacherDashboard/SectionActionDropdown.jsx

```diff
@@ -94,17 +106,30 @@ class SectionActionDropdown extends Component {
   };
 
   render() {
-    const {sectionData} = this.props;
+    const {sectionData, userId} = this.props;
 
     return (
       <span>
         <QuickActionsCell type={'header'}>
-          <PopUpMenu.Item
-            onClick={this.onClickEdit}
-            className="edit-section-details-link"
-          >
-            {i18n.editSectionDetails()}
-          </PopUpMenu.Item>
+          {/* Note that this should be uncommented when
+          ready to launch.  Also, remove line 125 for launch */}
+          {/* {userId % 10 === 0 && (
+            <PopUpMenu.Item
+              href={this.editRedirectUrl(sectionData.id)}
+              className="edit-section-details-link"
+            >
+              {i18n.editSectionDetails()}
+            </PopUpMenu.Item>
+          )} */}
+          {/* {userId % 10 !== 0 && ( */}
+          {userId && (
```

Comment:
Thank you for leaving all this here and ready! Just confirming in my brain- there will always be a user id because you can only access the teacher dashboard if you're logged in!